packages/typescript/src/lib/TypescriptCompiler.ts

```diff
@@ -85,7 +85,7 @@ export class TypescriptCompiler {
     );
 
     const pathWithExt = path.resolve(normalizedRootDir, filenameWithExt);
-    return pathWithExt;
+    return pathWithExt.replace(/\\/g, '/');
```

Comment:
`var correctedPath = path.normalize(randomPath);`

Reply:
yep definitely a better solution, had to add it in two places but confirmed it works on Windows and Mac. Thanks for the review!